include/mesh.h

```diff
@@ -379,6 +387,8 @@ class Mesh {
   std::map<unsigned, std::shared_ptr<mpm::Material<Tdim>>> materials_;
   //! Logger
   std::unique_ptr<spdlog::logger> console_;
+  //! Neighbouring node-cell ids map
+  std::map<mpm::Index, std::set<mpm::Index>> node_cell_maps_;
```

Comment:
Let's not store this, as we don't use it for anything other than finding neighbours. Could you please rename to `map_node_cells_`

Reply:
@kks32 I am okay not to store this. Initially I stored it because the neighbours set in the cell has been used for the sharing-edge neighbour. I will refactor.